fe/src/main/java/org/apache/doris/load/loadv2/SparkLoadJob.java

```diff
@@ -742,6 +760,26 @@ private void unprotectedLogUpdateStateInfo() {
         Catalog.getCurrentCatalog().getEditLog().logUpdateLoadJob(info);
     }
 
+    // because current SparkLoadJob didn't persist job's tableId, so we need to get job's tableid from transaction mgr
+    private void fillJobTableIdSetWhenReplay() {
+        try {
+            DatabaseTransactionMgr databaseTransactionMgr =
```

Comment:
How about just save the table ids in SparkLoad, so that we don't need to get it from TxnMgr? Because of this logic, the coupling with TxnMgr may be very serious. we have to handle the problem such as "db not exist"...

Reply:
👌